__tests__/routes/groomers.test.js

```diff
@@ -0,0 +1,93 @@
+const request = require('supertest');
+const express = require('express');
+const Profiles = require('../../api/profile/groomerModel');
+const profileRouter = require('../../api/profile/groomerRouter');
+const server = express();
+server.use(express.json());
+
+jest.mock('../../api/profile/profileModel');
+// mock the auth middleware completely
+jest.mock('../../api/middleware/authRequired', () =>
+  jest.fn((req, res, next) => next())
+);
+
+describe('groomers router endpoints', () => {
+  beforeAll(() => {
+    // This is the module/route being tested
+    server.use(['/profile', '/groomers'], profileRouter);
+    jest.clearAllMocks();
+  });
+
+  describe('GET /groomers', () => {
+    it('should return 200', async () => {
+      Profiles.findAll.mockResolvedValue([]);
+      const res = await request(server).get('/groomers');
+
+      expect(res.status).toBe(200);
+      expect(res.body.length).toBe(0);
+      expect(Profiles.findAll.mock.calls.length).toBe(1);
+    });
+  });
+
+  describe('GET /groomers/:id', () => {
+    it('should return 200 when profile found', async () => {
+      Profiles.findById.mockResolvedValue({
+        id: '1',
+        name: 'Bob Smith',
+        email: 'bob@example.com',
+      });
+      const res = await request(server).get('/groomers/1');
+
+      expect(res.status).toBe(200);
+      expect(res.body.name).toBe('Bob Smith');
+      expect(Profiles.findById.mock.calls.length).toBe(1);
+    });
+
+    it('should return 404 when no user found', async () => {
+      Profiles.findById.mockResolvedValue();
+      const res = await request(server).get('/groomers/1');
+
+      expect(res.status).toBe(404);
+      expect(res.body.error).toBe('ProfileNotFound');
+    });
+  });
+
+  describe('POST /profile', () => {
+    it('should return 200 when profile is created', async () => {
```

Comment:
Nice job setting up your tests. Quick note, most POST's should a return a '201' upon successful creation, not '200'.